ivy_tests/test_ivy/test_functional/test_core/test_elementwise.py

```diff
@@ -57,10 +57,11 @@ def test_abs(*, dtype_and_x, test_flags, backend_fw, fn_name, on_device):
 @handle_test(
     fn_tree="functional.ivy.acosh",
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("float"),
+        available_dtypes=helpers.get_dtypes("float_and_complex"),
         min_value=1,
-        large_abs_safety_factor=4,
-        small_abs_safety_factor=4,
+        large_abs_safety_factor=2.1,
```

Comment:
You've tested with a relatively large number of examples before reducing the safety factors right? Sometimes hypothesis isn't able to find a failing example until we run with like `100` max_examples

Reply:
I also set the scale to `"log"` so the safety factors are actually bigger now (also tested just now with 100 examples per backend and everything passes except the gradient thing)

Reply:
perfect, thanks!